### ISSUE TITLE
Compatibility List

### ISSUE THREAD

**Issue by najmsheikh** [omikun/MyAppNap#13]
I think it'd be a good idea to consolidate a "Compatibility List" since napping some apps will cause them to hang indefinitely unless the script is killed and restarted. I use iTerm2 and made the mistake of doing the default "nap everything" setting. It napped even the calling shell so that caused some issues lol.

**Comment by omikun** [comment]
Good idea! I have a list going in the script, but I'll add iTerm2 to it too.
